controllers/k8ssandracluster_controller_test.go

```diff
@@ -17,7 +20,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/cluster"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
-	"testing"
 )
 
 func testK8ssandraCluster(ctx context.Context, t *testing.T) {
```

Comment:
We need to verify that the ReplicatedSecret is created.

We also need coverage in the e2e tests. It might be sufficient to check that the secrets are the same across clusters.